iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/algorithm.inl

```diff
@@ -70,8 +72,9 @@ inline constexpr bool doesContainType() noexcept
 }
 
 template <typename T>
-inline constexpr bool doesContainValue(const T) noexcept
+inline constexpr bool doesContainValue(const T v) noexcept
```

Comment:
```suggestion
inline constexpr bool doesContainValue(const T v IOX_MAYBE_UNUSED) noexcept
```

Reply:
Fixed in: iox-#1196-fix-generic-raii-and-helplets